### ISSUE TITLE
Issue in Bulwark's Ambry

### ISSUE THREAD

**Issue by AyothaKAT** [DarkKronicle/CommandPotential#12]
Could leave it, but the keys in the Ambry allow for other boss item picks which could allow a farm. It's whatever, but seemed like it was unplanned :O

**Comment by DarkKronicle** [comment]
Yeah I am planning on adding an option for this. Quite busy atm but hopefully I can get to it soon.
